sys/compat/cheriabi/cheriabi_misc.c

```diff
@@ -1769,9 +1769,9 @@ int
 cheriabi_ptrace(struct thread *td, struct cheriabi_ptrace_args *uap)
 {
 	union {
-		struct ptrace_io_desc_c piod;
+		struct ptrace_io_desc piod;
 		struct ptrace_lwpinfo pl;
-		struct ptrace_vm_entry_c pve;
+		kptrace_vm_entry_t pve;
```

Comment:
1. This is a pointless diff, CheriABI has `kptrace_vm_entry_t` typedef'd to `struct ptrace_vm_entry_c`, but the latter also explicitly gives intentionality.
2. Can we not ditch ptrace_vm_entry_c and the `kptrace_vm_entry_t ` typedef as part of this and make it always be `struct ptrace_vm_entry` anyway?